vault/resource_pki_secret_backend_role.go

```diff
@@ -196,6 +198,19 @@ func pkiSecretBackendRoleResource() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			consts.FieldAllowedURISansTemplate: {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Flag to indicate that `allowed_uri_sans` specifies a template expression (e.g. {{identity.entity.aliases.<mount accessor>.name}})",
+			},
+			consts.FieldAllowedWildcardCertificates: {
+				Type:        schema.TypeBool,
+				Required:    false,
+				Optional:    true,
+				Description: "Flag to allow wildcard certificates",
+				Default:     true,
```

Comment:
The default value returned by the Vault API is "true" so we must set the default here.